### DISCUSSION TITLE
Removing specific cookies from requests

### DISCUSSION THREAD

**Discussion by karolis-palsauskas-devoteam** [reduxjs/redux-toolkit#4402] [Q&A]
Looking for a way to exclude specific cookies from being attached to requests. Tried using middleware headers, but they seem to ignore anything I modify.

Also tried separating api slices and updating credentials to "include" and "same-origin"/"omit" to those slices that would require cookies be omitted, but... slices with included cookies run fine first time, but second time they omit or do same-origin (like inherit this option from other slices)

**Comment by phryneas** [comment]
I mean, cookies are added by your browser. If there's no way of doing this with `fetch`, there is also no way to do it with RTKQ, which just calls `fetch` :/
